src/utils.ts

```diff
@@ -266,3 +266,19 @@ export function searchLinkByRel(links: Link[], rel: string): Link[] {
   var regularExpression = new RegExp('\\b' + escapeRegExp(rel) + '\\b');
   return links.filter((l) => regularExpression.test(l.rel));
 }
+
+export function intlFormatNumber(lang: string, format: string, value: number) {
```

Comment:
Perhaps it would be wise to change the function number, so not to be mistaken by an intl function. That's what I thought at first 😅 Something like this or any other thing:
```suggestion
export function formatNumber(lang: string, format: string, value: number) {
```

Reply:
I don't get it, this is a function that is related with i18n. It will return a number formatted based on language.